tensorflow/core/kernels/cudnn_rnn_ops.cc

```diff
@@ -500,6 +500,9 @@ struct CudnnRnnModelShapes {
   int max_seq_length;
   int batch_size;
   int cell_num_units = 0;
+  // TODO: If you add new field to that structure, please take care of
```

Comment:
Please delete the `TODO` since that has a very specific meaning.  And s/that/this/